packages/devtools_app/lib/src/shared/status_line.dart

```diff
@@ -176,10 +186,18 @@ class StatusLine extends StatelessWidget {
           return child!;
         }
       },
-      child: Text(
-        'No client connection',
-        style: textTheme.bodyText2,
-      ),
+      child: isExtraNarrow
+          ? DevToolsTooltip(
+              message: 'No client connection',
```

Comment:
put 'No client connection' into a const since it is used twice

Reply:
Done, thanks!